components/brave_ads/core/internal/common/url/url_util_internal.cc

```diff
@@ -96,10 +96,11 @@ bool DoesETLDPlusOneContainWildcards(const GURL& url) {
   // containing a '?' are deemed invalid.
   constexpr char kUrlEncodedAsteriskWildcard[] = "%2A";
 
-  return base::Contains(
+  const std::string domain_and_registry =
       net::registry_controlled_domains::GetDomainAndRegistry(
-          url, net::registry_controlled_domains::INCLUDE_PRIVATE_REGISTRIES),
-      kUrlEncodedAsteriskWildcard);
+          url, net::registry_controlled_domains::INCLUDE_PRIVATE_REGISTRIES);
+
+  return base::Contains(domain_and_registry, kUrlEncodedAsteriskWildcard);
```

Comment:
domain_and_registry.contains ?

Reply:
std::string does not have .contains

Reply:
ah, yes. It is added in c++23: https://en.cppreference.com/w/cpp/string/basic_string/contains